es3.0/html/indexflat.php

```
@@ -5,6 +5,22 @@
     <!-- Don't include accord.js -->
 </head>
 <body>
+  <script>
+    window.onload = function () {
+      try {
+        let params = new URL(document.location.toString()).searchParams;
+        let subpage = params.get("subpage");
+        if (subpage !== null) { 
+          const links = document.querySelectorAll('a');
+          for (let link of links) {
+            if (link.textContent === subpage) {
+              link.click() 
+            }
+          }
+        }
+    } catch (error) { }
```

Very silently failure for this, could log something to the console.